backend/src/main/java/team/YongAndJoe/NewsTodayBackend/entity/User.java

```diff
@@ -29,6 +37,14 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    public String getConfirmpassword() {
```

Comment:
Do not need confirm password in backend. Only frontend needs to confirm password before making the request.